src/main.ts

```diff
@@ -884,7 +884,8 @@ export class Purchases {
    */
   public isSandbox(): boolean {
     return (
-      isSandboxApiKey(this._API_KEY) || isSimulatedStoreApiKey(this._API_KEY)
+      isWebBillingSandboxApiKey(this._API_KEY) ||
+      isSimulatedStoreApiKey(this._API_KEY)
```

Comment:
I've been thinking, it might be good to move this logic to the `api-key-helper` and use it in the `X-Is-Sandbox` header and possibly the `Backend` class calculation, which seems to only be used in tests...

Not directly related to this PR though, so can be done in a separate PR, or I can do it later.